packages/airnode-deployer/terraform/gcp/main.tf

```diff
@@ -250,9 +250,9 @@ module "oevGw" {
   }
   project = var.gcp_project
 
-  invoke_targets = [
-    module.signOevReq[0].function_name
-  ]
+  invoke_targets = {
```

Comment:
Yes, it broke the deployment for GCP. It was caused by an incorrect merge. The OEV gateway was written in the old format but the format was updated before it was merged. And since it was a new part of the file it didn't create conflicts.